README.md

```diff
@@ -60,12 +60,19 @@ Alternatively, you can manually edit Application.xml and then reload the Wowza a
   </Property>
 ```
 
-### To deploy plugin code to VM:
-
-- (ensure Wowza application on VM has the properties above)
-- (update version file)
-- (tag version on github)
-- (trigger jenkins build that creates versioned deployment jar artifact)
+### To deploy a new version of the plugin code to the VM:
+
+- Ensure the Wowza application on the VM has the properties mentioned above.
```

Comment:
my bad - I think this should say "module and properties" instead of just "properties".  Also, it's okay if jar is deployed without an application configured -- so maybe we should actually split "deploy" from "configure"